src/test/scala/ilc/feature/let/BetaReductionSuite.scala

```diff
@@ -83,8 +83,8 @@ extends FlatSpec
   val duplicating = lambda(Var("f", NatType =>: NatType), Var("expensive", NatType)) {
     case Seq(arg, expensive) => PlusNat ! (arg ! expensive) ! (arg ! expensive)
   } ! u ! (PlusNat ! 42 ! 84)
-  pretty(duplicating)
-  pretty(normalize(duplicating))
+  info("\n" + pretty(duplicating))
+  info("\n" + pretty(normalize(duplicating)))
```

Comment:
Indeed, the old code looks indeed broken (probably coming from a worksheet).
I was thinking of [show()](https://github.com/inc-lc/ilc-scala/blob/a8f616b7f44a9cb0092b95af314ec40047b69d13/src/test/scala/ilc/feature/let/Instantiations.scala#L8), which again was used for worksheets so it just returns the string.